.release-please-manifest.json

```diff
@@ -1 +1 @@
-{".":"0.51.4"}
+{".":"0.51.1"}
```

Comment:
it should work now.
But it looks like when we release patches - `Release-please` will update tag in this file and we will get same case.

But I could be wrong.

Reply:
Release Please will update this file from v0.51.1 to v0.52.0, which will be included in the main branch. So, the release PR after v0.52.0 will work properly.